Tests/LibCore/TestLibCoreMappedFile.cpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2021, sin-ack <sin-ack@protonmail.com>
```

Comment:
I don't think that's right :P

Reply:
It might be, I recognize at least some of these test cases (which means that they originate from preexisting Core::Stream tests).

Reply:
Yes, these are directly adopted from the File tests. Will update the copyright.